choco/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$ErrorActionPreference = 'Stop'; 
+$toolsDir    = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$packageName = 'tektoncd-cli'
+$url64       = 'https://github.com/tektoncd/cli/releases/download/v0.18.0/tkn_0.18.0_Windows_x86_64.zip' 
```

Comment:
May be we would like to update the version here.

Reply:
Its a placeholder, you can use it for any version from what i understood